### ISSUE TITLE
Wrong ndcg results when evaluating only one k

### ISSUE THREAD

**Issue by SimonCW** [karlhigley/ranking-metrics-torch#1]
Hi @karlhigley, I just noted this oddity while playing with your implementation. Unfortunately, I didn't have time to further investigate the issue.

Observed behaviour: 
```python
import torch
from ranking_metrics_torch.cumulative_gain import ndcg_at

true = torch.tensor([[10, 0, 0, 1, 5]])
scores = torch.tensor([[0.1, 0.2, 0.3, 4, 70]])
ks = torch.tensor([3])
ndcg_at(ks, scores=scores, labels=true)
```
Returns
`tensor([[1.]], dtype=torch.float64)`

However adding another k to evaluate returns the correct ndcgs:
```python
ks = torch.tensor([3, 5])
ndcg_at(ks, scores=scores, labels=true)
```
Returns:
`tensor([[0.4124, 0.6957]], dtype=torch.float64)`

**Comment by karlhigley** [comment]
Hmm, weird, I'll take a look and see if I can figure it out.

**Comment by karlhigley** [comment]
@SimonCW Turns out that it only works when the largest k matches the length of the labels (whether or not you pass multiple ks.) The normalizing gains should be computed with the full list of labels, but were being computed with the topk labels instead. :sweat_smile:
